Gems/EMotionFX/Code/EMotionFX/Source/Actor.cpp

```diff
@@ -1443,6 +1443,13 @@ namespace EMotionFX
             {
                 m_skinToSkeletonIndexMap = ConstructSkinToSkeletonIndexMap(m_skinMetaAsset);
             }
+            else
+            {
+                AZ_Error(
+                    "Actor", false,
+                    "Actor finalization: Skin meta asset was expected to be ready but is not ready yet.  Cannot complete finalizing actor %s",
```

Comment:
is there an action the user can do to fix this error?

Reply:
I don't believe so, it's a system/code error.  Perhaps it should be an assert instead?  I'm on the fence because the system can still function without this, the actor just won't render, so that felt like an error to me.  Thoughts?

I put this in because I spent a lot of time tracking down this problem in the first place, having an error pop up would have made it a lot faster to find the issue.